src/components/InfoTable/InfoTable.tsx

```diff
@@ -24,8 +29,10 @@ function InfoTable({ header, rows, ...boxProps }: TInfoTable) {
       <TableContainer component={Paper}>
         <MuiTable sx={{ minWidth: { sm: 650 } }} aria-label="table">
           <TableBody>
```

Comment:
My implementation of the second policy holder table just adds a new policy holder to the second table every time the user clicks the create button. This was giving console warnings that rows in the table had the same keys, so I changed the key of the row here to have the index as well so the keys would not be identical